susemanager/src/mgr_bootstrap_data.py

```diff
@@ -247,117 +66,12 @@
 
 PKGLIST15_Z = []
 
-PKGLISTUBUNTU1604 = [
-    "libsodium18",
-    "dctrl-tools",
-    "libzmq5",
-    "python-chardet",
-    "python-croniter",
-    "python-crypto",
-    "python-dateutil",
-    "python-enum34",
-    "python-ipaddress",
-    "python-jinja2",
-    "python-markupsafe",
-    "python-minimal",
-    "python-msgpack",
-    "python-openssl",
-    "python-pkg-resources",
-    "python-psutil",
-    "python-requests",
-    "python-six",
-    "python-systemd",
-    "python-tornado",
-    "python-tz",
-    "python-urllib3",
-    "python-yaml",
-    "python-zmq",
-    "python-pycurl",
-    "salt-common",
-    "salt-minion",
-    "dmidecode",
-]
 
 PKGLISTUBUNTU1804 = [
-    "dctrl-tools",
-    "javascript-common",
-    "libjs-jquery",
-    "libjs-sphinxdoc",
-    "libjs-underscore",
-    "libnorm1",
-    "libpgm-5.2-0",
-    "libpython3.6-stdlib",
-    "libpython3.6-minimal",
-    "libpython3.6-stdlib",
-    "libsodium23",
-    "libzmq5",
-    "python3",
-    "python3-apt",
-    "python3-asn1crypto",
-    "python3-certifi",
-    "python3-cffi-backend",
-    "python3-chardet",
-    "python3-croniter",
-    "python3-crypto",
-    "python3-pycryptodome",
-    "python3-cryptography",
-    "python3-dateutil",
-    "python3-idna",
-    "python3-jinja2",
-    "python3-markupsafe",
-    "python3-minimal",
-    "python3-msgpack",
-    "python3-openssl",
-    "python3-pkg-resources",
-    "python3-psutil",
-    "python3-requests",
-    "python3-singledispatch",
-    "python3-six",
-    "python3-systemd",
-    "python3-tornado",
-    "python3-tz",
-    "python3-urllib3",
-    "python3-yaml",
-    "python3-zmq",
-    "python3.6",
-    "python3.6-minimal",
-    "salt-common",
-    "salt-minion",
-    "dmidecode",
-    "bsdmainutils",
-    "debconf-utils",
-    "iso-codes",
-    "python-apt-common",
-    "python3-distro",
-    "python3-gnupg",
-    "gnupg",
-    "python3-immutables",
-    "python3-contextvars",
-    "python3-looseversion",
-    "python3-jmespath",
     "venv-salt-minion",
 ]
```

Comment:
Ubuntu 18.04 is EOL. I think we can drop it  completely. sorry. I can't add a  suggestion with a full drop